great_expectations_cloud/agent/agent.py

```diff
@@ -134,17 +135,17 @@ class GXAgent:
     def __init__(self: Self):
         agent_version: str = self.get_current_gx_agent_version()
         great_expectations_version: str = self._get_current_great_expectations_version()
-        print(f"GX Agent version: {agent_version}")
-        print(f"Great Expectations version: {great_expectations_version}")
-        print("Initializing the GX Agent.")
+        LOGGER.info(f"GX Agent version: {agent_version}")
+        LOGGER.info(f"Great Expectations version: {great_expectations_version}")
+        LOGGER.info("Initializing the GX Agent.")
         self._config = self._get_config()
-        print("Loading a DataContext - this might take a moment.")
+        LOGGER.info("Loading a DataContext - this might take a moment.")
```

Comment:
I think generally the types of messages that are intended to determine when a section of code starts and ends should be at `debug` level. They're generally only useful when debugging